examples/features/standard/interceptor-client/pom.xml

```diff
@@ -60,7 +55,7 @@ under the License.
                      <goal>create</goal>
                   </goals>
                   <configuration>
-                     <libList><arg>org.apache.activemq.examples.broker:interceptor:${project.version}</arg></libList>
+                     <libList><arg>org.apache.activemq.examples.broker:interceptor-client:${project.version}</arg></libList>
```

Comment:
Should this be needed here at all, if its actually a client-side example? This bit being for creating the broker used. So cant this just be removed?

Reply:
You are right, I removed it.